rust/template/cmd_parser/parse.rs

```diff
@@ -406,6 +414,28 @@ fn test_string() {
     );
 }
 
+named!(serialized_val<&[u8], Record>,
+    do_parse!(
+        tag!("@")
```

Comment:
Do you want to document this particular format?
How does one register a new deserializer?

Reply:
yes, there are a couple of other recent undocumented extensions.  Let will submit a separate documentation PR.